zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -172,62 +43,16 @@
     </navigation>
   </page>
 
-
   <!-- Admin -->
   <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
   <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
```

Comment:
Done in
https://github.com/zanata/zanata-server/commit/825e320af3cad8393031a6058a681b0e6736b944

On Thu, Sep 24, 2015 at 3:59 PM, Carlos Munoz <notifications@github.com>
wrote:

> In zanata-war/src/main/webapp/WEB-INF/pages.xml
> <https://github.com/zanata/zanata-server/pull/959#discussion_r40287936>:
>
> > @@ -172,62 +43,16 @@
> >      </navigation>
> >    </page>
> >
> > -
> >    <!-- Admin -->
> >    <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
> >    <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
>
> I have no problem with adding the check to every admin xhtml page; in fact
> that's the approach we are taking with all other pages.
> In fact I prefer that to a servlet filter as the it's clear what actions
> are being performed by looking at the view.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/959/files#r40287936>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com